source/directives/uiTreeNode.js

```diff
@@ -354,180 +411,175 @@
                     treeScope.resetEmptyElement();
                     treeScope = null;
                   }
+
+                  //Reset parent to source parent.
+                  dragInfo.resetParent();
                 }
 
-                // move horizontal
-                if (pos.dirAx && pos.distAxX >= config.levelThreshold) {
```

Comment:
@jacobscarter I think this block (and a couple under it) are related to the regression. The horizontal movement referred to is related to horizontal mouse movement to change the nesting of a tree item, not the obsolete 'horizontal tree' concept.